<launch>

  <node   
      name="calc_grasppoints_svm_action_server" 
    pkg="haf_grasping" 
    type="calc_grasppoints_action_server">        
  </node>
  <node   
      name="calc_grasppoints_svm_action_client" 
    pkg="haf_grasping" 
    type="calc_grasppoints_client_sim"
    output="screen">        
  </node>

  <rosparam param="opening_width_scale_marker">2</rosparam>
  <rosparam param="gripper_width">1</rosparam>


</launch>